packages/jest-config/src/__tests__/normalize.test.js

```diff
@@ -1440,3 +1431,11 @@ describe('moduleFileExtensions', () => {
     ).toThrowError("moduleFileExtensions must include 'js'");
   });
 });
+
+describe('Defaults', () => {
+  it('should be accepted by normalize', () => {
+    normalize({...Defaults, rootDir: '/root'}, {});
```

Comment:
`rootDir` is added by `readConfig`

Reply:
Might want to preserve that information 😄 
```suggestion
	// rootDir is added by readConfig
    normalize({...Defaults, rootDir: '/root'}, {});
```

Reply:
I don't see a reason, it's used like this all over this test file